script.js

```diff
@@ -770,3 +770,20 @@ function canVisitCommunity(role, userSegmentExists) {
 /**** END GET USER DATA ****/
 
 $(() => {});
+
+document.addEventListener('DOMContentLoaded', function() {
+  var acc = document.getElementsByClassName("accordion");
+  var i;
+
+  for (i = 0; i < acc.length; i++) {
```

Comment:
Here, you can just put in the `var` on the same line like this:

```
for (var i = 0; i < acc.length; i++) { ... }
```

Or you should be able to use `forEach` like this:

```
acc.forEach(function (item, index) {
    item.addEventListener("click", function () {
        ...
    });
})
```

Reply:
I have tried both of your suggestions and it doesn't seem to work.
I'm not very well versed in js code, so I'm not sure how to make your suggestions work.
Using notepad++ to edit code, perhaps there are better debugging tools?